lib/vmdb/plugins.rb

```diff
@@ -77,7 +77,11 @@ def provider_plugins
     def asset_paths
       @asset_paths ||= begin
         require_relative 'plugins/asset_path'
-        map { |engine| AssetPath.new(engine) if AssetPath.asset_path?(engine) }.compact
+        map do |engine|
+          in_bundler_gems = engine.root.expand_path.to_s.start_with?(Bundler.install_path.expand_path.to_s)
```

Comment:
Minor, but this logic can live right in the AssetPath class...then you don't need to modify the params to initialize.  Since the information comes from the engine itself, and we are already passing the engine in, then it's not a problem for it to be in there.

Reply:
Makes sense :+1: